templated_email/__init__.py

```diff
@@ -1,10 +1,10 @@
 from django.conf import settings
 from django.utils.module_loading import import_string
-from templated_email.backends.vanilla_django import TemplateBackend
 
 import six
-import warnings
-warnings.filterwarnings('error', 'django.utils.importlib')
+
+from templated_email.backends.vanilla_django import TemplateBackend
+from templated_email.utils import InlineImage  # noqa
```

Comment:
it's for flake8 to ignore it not being used in the file. But we want it there to allow `from templated_email import InlineImage`.